skimage/restoration/_denoise.py

```diff
@@ -1,12 +1,13 @@
 # coding: utf-8
 import numpy as np
+from math import ceil
 from .. import img_as_float
 from ..restoration._denoise_cy import _denoise_bilateral, _denoise_tv_bregman
 from .._shared.utils import _mode_deprecations
 import warnings
 
 
-def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
+def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
                       bins=10000, mode='constant', cval=0, multichannel=True):
```

Comment:
This will support positional arguments as well as warn users who were using the legacy kwarg. The only downside is that there are two separate kwargs for the same functionality. To mask this to the user, we could hide the legacy `sigma_color` from the user in `**kwargs` and leave it off the docstring.